### DISCUSSION TITLE
插件已经识别但是不显示

### DISCUSSION THREAD

**Discussion by dysf888** [metatube-community/jellyfin-plugin-metatube#153] [Q&A]
![image](https://user-images.githubusercontent.com/47450409/228172286-43a8836c-eed6-4308-9c9a-d51563c7ead7.png)
![image](https://user-images.githubusercontent.com/47450409/228172328-2c5895ab-2e96-432c-8b3a-e8251a3f7052.png)
![image](https://user-images.githubusercontent.com/47450409/228172375-d11d4e4a-c88a-46b0-950d-c8536dd70da3.png)
![image](https://user-images.githubusercontent.com/47450409/228172585-4a746dd0-8323-404a-9cfd-a9ff6d2cc864.png)
![image](https://user-images.githubusercontent.com/47450409/228172607-000f8602-fc8d-4b35-b515-1489f15cb62e.png)
server重启过多次 无用

**Comment by xjasonlyu** [comment]
<img width="397" alt="Screenshot 2023-03-28 at 4 20 51 PM" src="https://user-images.githubusercontent.com/28824352/228174580-883e9f50-f14b-45e2-8a87-41b7b679fee4.png">

**Reply by dysf888** [reply]
thanks
